dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -74,7 +74,7 @@ protected String parseCommandText(String commandText) {
                     paramName = "";
                 String dataType = rs.getString(2);
                 DbParameterAccessor dbp = createDbParameterAccessor(
-                        paramName,
+                        '"' + paramName + '"',
```

Comment:
Right. I think the current change is well isolated (only plsql code is affected) and is working good enough. We can go with it as initial version and think of possible refactoring later.